ui/js/graphs.js

```diff
@@ -738,6 +741,24 @@ perf.controller('TestChooserCtrl', function($scope, $modalInstance, $http,
   var testArray = [];
   var series = [];
   $scope.addTestData = function () {
+    if ($scope.addedTestList.length > 6) {
+      var a = window.confirm('WARNING: You are about to load ' + $scope.addedTestList.length +
+                           ' tests to the graph.\n' +
+                           'Loading more than 6 is not recommended. Do it anyway?');
+      if (a == true) {
+        addTestToGraph();
+      }
+    } else if ($scope.addedTestList.length + testCounter > 6) {
+      var a = window.confirm('WARNING: You have ' + testCounter + ' tests in the graph already' +
+                          ' and You are about to load ' + $scope.addedTestList.length + ' more test.\n'
```

Comment:
A few problems:

* The capitalization is wrong here ("and You are...").
* tests should be plural at the end ("you are about to load X more tests").

I don't think we need to spell things out in so much detail for the user. It should be pretty obvious what's going on already. Let's just say something simple like 'WARNING: Displaying more than 6 graphs at the same time is not recommended. Do it anyway?' That way you can implement the simplification I suggest below.